typescript-requiring-type-checking.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
I guess it is a generated file; you have to put it to `.gitignore`.

Reply:
Good catch, let me fix it.

Reply:
Done.